libs/viewer/src/Settings.cpp

```diff
@@ -646,6 +651,20 @@ void applySettings(Engine* engine, const ViewerOptions& settings, Camera* camera
         camera->setFocusDistance(settings.cameraFocusDistance);
     }
     engine->setAutomaticInstancingEnabled(settings.autoInstancingEnabled);
+
+    // Eyes are rendered from left-to-right, i.e., eye 0 is rendered to the left side of the
+    // window.
+    // For testing, we want to render a side-by-side layout so users can view with
+    // "cross-eyed" stereo.
+    // For cross-eyed stereo, Eye 0 is really the RIGHT eye, while Eye 1 is the LEFT eye.
```

Comment:
Just curious, is there any reason why we didn't match the order of the eyes on the screen with the order of the array?

Reply:
So in fact we do, and maybe my comment is a little confusing. But eye 0 is always rendered to the left side of the screen, and eye 1 is rendered to the right side:

```
0 | 1
```

What this comment alludes to, is the fact that to render [cross-eyed stereo](https://www.kula3d.com/how-to-use-the-cross-eyed-method), the left and right eyes need to be flipped.